storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -2096,6 +2096,14 @@ char *make_argv(char *buf, size_t len, int argc, char **argv)
 				strlen("--encrypt_key")) == 0) {
 			arg = "--encrypt_key=...";
 		}
+		if (strncmp(*argv, "--transition-key",
+				strlen("--transition-key")) == 0) {
```

Comment:
strlen() should ideally be aligned to *argv.. but it looks like old code is doing wrong alignment too..

So either fix all strlen alignment or leave it as it is.. 
choice is yours :)

Reply:
I will leave it as it for now.